big_tests/tests/push_http_SUITE.erl

```diff
@@ -0,0 +1,279 @@
+-module(push_http_SUITE).
```

Comment:
@bartekgorny @erszcz Did you noticed that this SUITE is not even run on travis? 🙈  It's missing in `default.spec`. I wonder if it pass :) I'd like to see a new PR with this SUITE added to default spec. But let's merge it only after #2101 is merged.